screens/ViewDetails/ViewEventScreen/ViewEventScreen.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { View, Text, TouchableOpacity, Alert } from 'react-native'
+import { RouteProp, useRoute } from '@react-navigation/native'
+import { styles } from './styles'
+import { Event } from '../../../types/Event'
+import { useEffect, useState } from 'react'
+import LoadingScreen from '../../Loading/LoadingScreen'
+import { viewDetailsStyles } from '../ViewDetailsStyles'
+import MapView, { Marker, PROVIDER_GOOGLE, Point } from 'react-native-maps'
+import { globalStyles } from '../../../assets/global/globalStyles'
+import ProfilePicture from '../../../components/ProfilePicture/ProfilePicture'
+
+const dummyPoint: Point = {
+    x : 40.712776,
+    y : -74.005974
+}
+
+const dummyEvent: Event = {
+    uid: "1",
+    title: "Title of the event right at the start",
+    location: "Central Park",
+    point: dummyPoint,
+    date: new Date(), // current date and time
+    description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Dolor morbi non arcu risus quis. Mattis molestie a iaculis at. Tristique risus nec feugiat in fermentum posuere urna. Quisque sagittis purus sit amet volutpat consequat mauris nunc. Sapien eget mi proin sed libero. Condimentum mattis pellentesque id nibh tortor id aliquet lectus proin. Suspendisse faucibus interdum posuere lorem. In mollis nunc sed id semper risus. Amet consectetur adipiscing elit ut aliquam purus. Integer enim neque volutpat ac.    ",
+    imageUrl: ""
+}
+
+type RootStackParamList = {
+    ViewEvent: {
+        // Here we're getting a uid and not an event because it is possible that the app opens on this screen (by QR)
+        uid: string;
+    }
+}
+  
+type ViewEventScreenRouteProps = RouteProp<RootStackParamList, "ViewEvent">
+
+const ViewEventScreen = () => {
+  const { uid } = useRoute<ViewEventScreenRouteProps>().params
```

Comment:
I also think again, that you do not need a root stackParamList, but just the ViewEventScreenRouteProps. You can maybe take a look at how in #141 the event was passed as a props to the MapScreen. Again, I may be mistaken, so tell me your way of thinking please.

Reply:
It's the same reason as explained above. The implementation in #141 way actually fixed in a later PR in order to remove the typescript error. You can look at the code of the MapScreen in main, you'll find the same implementation.